src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const listSalary = [...document.querySelectorAll('li')];
+
+function parseSalary(salaryStr) {
+  return parseInt(salaryStr.replace(/[$,]/g, ''), 10);
+}
+
+function sortList(list) {
+  return list.sort((a, b) => {
+    const salaryA = parseSalary(a.getAttribute('data-salary'));
+    const salaryB = parseSalary(b.getAttribute('data-salary'));
+
+    return salaryB - salaryA;
+  });
+}
```

Comment:
The 'sortList' function could be more efficient. Right now, it is parsing each salary twice: once when comparing in the sort and once when creating the employee object. It would be more efficient to parse the salaries once, store them, and then use the stored values.